corehq/apps/export/static/export/js/download_data_files.js

```diff
@@ -12,7 +15,10 @@ hqDefine("export/js/download_data_files", function() {
      * @param url: The URL to be copied to the clipboard
      * @param textareaElem: A hidden <textarea> element containing the URL
      */
-    var copyDataFileUrl = function (url, textareaElem) {
+    var copyDataFileUrl = function () {
+        var url = window.location.origin + initialPageData.reverse('download_data_file',$(this).data("id"), $(this).data("name"));
```

Comment:
I don't think the `window.location.origin` here & below is necessary - the old `{% url ... %}` code would have been a relative url.

Reply:
it looks `window.location.origin` is required here. because earlier also `{% url ..%}` was used with `{{ url_base }}`. I think which also makes sense because this function copies the url to clipboard. and pasting url without base url would be meaningless

Reply:
Oops, I missed that - thanks for the explanation.